libraries/AP_Baro/AP_Baro_SPx06.h

```diff
@@ -2,23 +2,27 @@
 
 #include "AP_Baro_Backend.h"
 
-#if AP_BARO_SPL06_ENABLED
+#if AP_BARO_SPx06_ENABLED
 
 #include <AP_HAL/AP_HAL.h>
 #include <AP_HAL/Device.h>
 #include <AP_HAL/utility/OwnPtr.h>
 
-#ifndef HAL_BARO_SPL06_I2C_ADDR
- #define HAL_BARO_SPL06_I2C_ADDR  (0x76)
+#ifndef HAL_BARO_SPx06_I2C_ADDR
+ #define HAL_BARO_SPx06_I2C_ADDR  (0x76)
 #endif
-#ifndef HAL_BARO_SPL06_I2C_ADDR2
- #define HAL_BARO_SPL06_I2C_ADDR2 (0x77)
+#ifndef HAL_BARO_SPx06_I2C_ADDR2
+ #define HAL_BARO_SPx06_I2C_ADDR2 (0x77)
 #endif
 
-class AP_Baro_SPL06 : public AP_Baro_Backend
+class AP_Baro_SPx06 : public AP_Baro_Backend
 {
 public:
-    AP_Baro_SPL06(AP_Baro &baro, AP_HAL::OwnPtr<AP_HAL::Device> dev);
+	enum SPx06 {
+			BARO_SPL06 = 0,
+			BARO_SPA06 = 1,
+	};
```

Comment:
```suggestion
	enum class Type {
	                UNKNOWN,
			SPL06,
			SPA06,
	};
```

... can do away with the is_spl06 and is_spa06 booleans after this change